src/python/pants/backend/python/dependency_inference/scripts/BUILD

```diff
@@ -3,13 +3,11 @@
 
 resource(
     name="dependency_parser",
-    source="dependency_parser_py",
+    source="dependency_parser.py",
 )
 
-# Also expose scripts as python sources so they get formatted/linted/checked.
+# Also expose script as python sources so it gets formatted/linted/checked.
 python_source(
     name="dependency_parser_source",
-    source="dependency_parser_py",
-    # This is run with Python 2.7 and 3.5+, so we shouldn't be running pyupgrade.
-    # skip_pyupgrade=True,
```

Comment:
Either we enable pyupgrade, in which case we can add this in uncommented, or we don't, in which case this can't be uncommented. But having it commented is the worst of both worlds.

Reply:
If we `pyupgrade` it tests will fail, so :ship: